app.rb

```diff
@@ -0,0 +1,145 @@
+require './student'
+require './teacher'
+require './book'
+require './rental'
+
+class App
+  def initialize
+    @people = []
+    @books = []
+    @rentals = []
+  end
+
+  def show_menu
+    puts "\n\nWelcome to School Library App!\n\n"
+    puts 'Please choose an option by entering a number:'
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
+    puts '6 - List all rentals for a given person id'
+    puts "7 - Exit\n\n"
+    gets.chomp
+  end
+
+  def list_people
+    puts 'List of people:'
+    @people.each do |person, index|
+      puts "#{index}) - Name: #{person.name} ID: #{person.id} Age: #{person.age}"
+    end
+  end
+
+  def list_books
+    puts 'List of books:'
+    @books.each do |book, index|
+      puts "(#{index}) - Author: #{book.author} Title: #{book.title}"
+    end
+  end
+
+  def create_student
+    puts 'Enter student name:'
+    name = gets.chomp
+    puts 'Enter student age:'
+    age = gets.chomp.to_i
+    puts 'Enter student classroom (Grade 1, Grade 2, etc):'
+    classroom = gets.chomp
+    puts 'Enter student parent permission (Y/N):'
+    parent_permission = gets.chomp.to_s == 'Y'
+    @people << Student.new(age, name, parent_permission, classroom)
+    puts "\n"
+    puts "Student #{name} created!"
+  end
+
+  def create_teacher
+    puts 'Enter teacher name:'
+    name = gets.chomp
+    puts 'Enter teacher age:'
+    age = gets.chomp.to_i
+    puts 'Enter teacher specialization:'
+    specialization = gets.chomp
+    @people << Teacher.new(name, age, specialization)
+    puts "\n"
+    puts "Teacher #{name} created!"
+  end
+
+  def create_rental
+    puts 'Enter person index from list below (Not ID):'
+    list_people
+    person_id = gets.chomp.to_i
+    person = @people[person_id]
+    if check_permissom(person)
+      puts 'Enter book index from list below (Not ID):'
+      list_books
+      book_id = gets.chomp.to_i
+      puts 'Enter rental date (YYYY-MM-DD):'
+      date = gets.chomp
+      @rentals << Rental.new(date, @people[person_id], @books[book_id])
+      puts "\n"
+      puts 'Rental created!'
+    else
+      puts 'You do not have permission to rent books!'
+    end
+  end
```

Comment:
- [ ] When creating a rental ı can't see the index number(inside bracket) next to the name. Kindly add it.

![image](https://user-images.githubusercontent.com/38635158/186822866-d86dc03f-469a-4de6-8a4d-5f7554b304c8.png)